components/text-field/src/atoms/input-messaging/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import LabelText from '@govuk-react/label-text';
+import ErrorText from '@govuk-react/error-text';
+import HintText from '@govuk-react/hint-text';
+
+const InputMessaging = ({
```

Comment:
@stevesims - I've abstracted out the input messaging to an Atom, which will be promoted to a component when building the other form fields.